CanvasPlusPlayground/Views/AsyncAttributedText.swift

```diff
@@ -9,18 +9,18 @@ import SwiftUI
 
 struct AsyncAttributedText: View {
     let htmlText: String
-    @State var announcementText:NSAttributedString? = nil
-    
+    @State var announcementText: NSAttributedString? = nil
+
     var body: some View {
         Group {
             if let announcementText {
-                AttributedText(announcementText)
+                Text(AttributedString(announcementText))
```

Comment:
@iWatger got rid of the `AttributedString` view representable you were using and now using the native SwiftUI one. It appears slightly different but still functionally the same.